python/ray/includes/gcs_client.pxi

```diff
@@ -559,6 +640,29 @@ cdef convert_get_all_job_info(
     except Exception as e:
         return None, e
 
+cdef convert_get_all_actor_info(
+        CRayStatus status, c_vector[CActorTableData]&& c_data):
+    # -> Dict[ActorID, gcs_pb2.ActorTableData]
+    cdef c_string b
+    try:
+        check_status_timeout_as_rpc_error(status)
+        actor_table_data = {}
+        for c_proto in c_data:
+            b = c_proto.SerializeAsString()
+            proto = gcs_pb2.ActorTableData()
+            proto.ParseFromString(b)
+            actor_table_data[ActorID.from_binary(proto.actor_id)] = proto
```

Comment:
So this is the extra overhead introduced in this PR since the old way doesn't go through c++ at all?

Reply:
I'd say yes since there's one more conversion happening. But Actor Info is not big anyway since the bulk part is split out to the ActorTask table.